lib/internal/bootstrap/realm.js

```diff
@@ -214,7 +214,7 @@ const publicBuiltinIds = builtinIds
   );
 // Do not expose the loaders to user land even with --expose-internals.
```

Comment:
The comment here already explained that it should not be exposed even with `--expose-internals`

Reply:
If a user is using the flag `--expose-internals`, than they are already using debug/experimental/internal features, so what makes this special?

Reply:
It looks like an error in the comment instead. It should be updated to `Be careful not to expose this even if --expose-internals is used`.

> than they are already using debug/experimental/internal features, so what makes this special?

Because this allows modification to the module loader itself and how other internal modules load each other, which can cause mayhem. In the unlikely event that someone uses `--expose-internals` and count on it, we can afford the compatibility burden to some extent, but the cost of maintaining this would be way too high to even consider exposing this to the user land behind a flag.

Reply:
Okay sounds good. I'll make a separate PR to update the comment